pkg/mcs/exportspec_test.go

```diff
@@ -78,25 +97,56 @@ func TestCompatibility(t *testing.T) {
 		other      *mcs.ExportSpec
 		compatible bool
 		field      string
+		mutate     func(mcs.ExportSpec) *mcs.ExportSpec
```

Comment:
I added the mutate function to edit the samples instead of inserting two samples, as you did in the Marshal tests.
Each time the mutate function changed the sample according to the test needed.